src/@ionic-native/plugins/zeroconf/index.ts

```diff
@@ -0,0 +1,132 @@
+import { Injectable } from '@angular/core';
+import { Cordova, Plugin } from '@ionic-native/core';
+import { Observable } from 'rxjs/Observable';
+
+export interface ZeroconfService {
```

Comment:
Yes, it's complicated ;-)
Thanks, I really think it is easier for people who actually know/use zeroconf that way, even if "ZeroConf" looks more natural first!